python/ray/serve/_private/replica_scheduler/replica_wrapper.py

```diff
@@ -16,6 +16,10 @@
 from ray.serve._private.utils import JavaActorHandleProxy
 from ray.serve.generated.serve_pb2 import RequestMetadata as RequestMetadataProto
 
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
should use `logging.getLogger(SERVE_LOGGER_NAME)` for consistency w/ rest of the codebase

i'm actually not sure anymore if this practically makes a difference (there have been a few log refactors over the years)